Support/Testing/Blacklists/ds2/x86_64.blacklist

```diff
@@ -1,2 +0,0 @@
-xfail
```

Comment:
Does that delete the file or just empty it? The github UI doesn't make it clear.

Reply:
The file itself is removed.

Reply:
It's deleted.